output/mapping/eforms/guidance.yaml

```diff
@@ -6548,7 +6548,7 @@
 
         - Ensure its `.id` is set. The `.id` can be any value guaranteed to be unique within the scope of the procedure. For example, it can be set to a [version 4 UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier), or it can be assigned sequentially across all notices for this procedure (the first notice sets it to "1", the second to "2", etc.).
         - Map this field to its `.rationaleClassifications.id`.
-        - Set its `.rationaleClassifications.scheme` to 'modification justification'.
+        - Set its `.rationaleClassifications.scheme` to 'eu-change-corrig-justification'.
```

Comment:
Need to also reword these bullets to treat rationaleClassifications as an array instead of as an object.

Reply:
Done in https://github.com/open-contracting/european-union-support/pull/254/commits/ae5eb0fe4efd2c11c78be75605ddb4efc5c17d58